lib/scores.js

```diff
@@ -0,0 +1,114 @@
+const Async = require('async');
+const level = require('../lib/level.js');
```

Comment:
@Danwhy bearing in mind that this ties the _score_ to a single server I would _recommend_ you consider asking @jamieu if we can use Redis (ElastiCache): http://docs.aws.amazon.com/AmazonElastiCache/latest/UserGuide/Clusters.Create.Redis.CON.html to ensure _horizontal scalability_ ...